### ISSUE TITLE
Screen-reader speech is not stopped on menu exit

### ISSUE THREAD

**Issue by munawarb** [munawarb/Three-D-Velocity#100]
Screen-reader speech is not interrupted when a user presses ENTER on a menu choice. The speech should be stopped once a menu loop exits.

**Comment by burakyuksek** [comment]
Hello,
Can the same thing be done for status screen so it interrupts when it's about say another status item? I'm putting the screen reader on sleep so that's why it matters.

**Comment by munawarb** [comment]
This is a great point. I'll look into it.
